Gemfile

```diff
@@ -25,6 +25,12 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 
+# Serialize the data
+gem 'fast_jsonapi'
```

Comment:
This gom looks like is not maintained anymore by netflix https://github.com/Netflix/fast_jsonapi/issues/462

Reply:
Fixed